permission_handler/example/pubspec.yaml

```
@@ -2,7 +2,7 @@ name: permission_handler_example
 description: Demonstrates how to use the permission_handler plugin.
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```

Update this to the NNBD stable version:

```suggestion
  sdk: ">=2.12.0-259.9.beta <3.0.0"
```